cmd/meroxa/turbine_cli/utils.go

```diff
@@ -17,17 +17,25 @@ import (
 )
 
 type AppConfig struct {
-	Name     string `json:"name"`
-	Language string `json:"language"`
+	Name        string            `json:"name"`
+	Environment string            `json:"environment"`
+	Pipeline    string            `json:"pipeline"`
```

Comment:
I don't think this should be here.

Reply:
yeah, if it's gone everywhere else already, don't need it here. 
```suggestion
```